.github/workflows/buildAndTest.yml

```diff
@@ -34,8 +34,10 @@ jobs:
         id: cache-llvm
         uses: actions/cache@v2
         with:
-          path: llvm
-          key: ${{ runner.os }}-llvm-sharedlibs-${{ steps.get-llvm-hash.outputs.hash }}
+          path: |
+            llvm/build
+            llvm/install
+          key: ${{ runner.os }}-llvm-nocode-sharedlibs-${{ steps.get-llvm-hash.outputs.hash }}
```

Comment:
I've considered just removing one of these cache busting strings (i.e. `sharedlibs-`) instead of adding more :p Maybe next time we can shrink the key.

Reply:
Ha! Yeah...